### ISSUE TITLE
The "time" result should be taken from across multiple requests and not a single request

### ISSUE THREAD

**Issue by motin** [kenjis/php-framework-benchmark#37]
Currently, [the "time" results is taken from a single request](https://github.com/kenjis/php-framework-benchmark/blob/963f129a124bb6ec2f8c46d731aa2ee704b34cc6/benchmarks/_functions.sh#L10-L14)

I suggest we use the result that ab already reports "Time per request:       78.445 [ms] (mean)" which is the mean based on all requests. It would also be interesting to use ab's "Time per request:       7.845 [ms] (mean, across all concurrent requests)" for comparison reasons.

**Comment by kenjis** [comment]
Yeah, I also want to fix the problem. Once is too less, and the value is not stable.
But ab's time is not the same.

**Comment by motin** [comment]
What about taking ab's time for each framework and then subtracting ab's time for "no-framework" requests? That would give the actual real-world additional bootstrap cost for the framework. Related discussion: https://github.com/kenjis/php-framework-benchmark/issues/34
